docs/docs/developers/reference/environment_reference/cli_wallet_reference.md

```diff
@@ -41,6 +41,64 @@ You can create arbitrary aliases with the `alias` command. For example `aztec-wa
 
 :::
 
+## Paying Fees
+
+
+import { Why_Fees, CLI_Fees } from '/components/snippets';
+
+<Why_Fees />
+
+Below are all the payment methods available to pay transaction fees on Aztec, starting with the simplest.
+
+### Sponsored Fee Paying Contract
+
+This is a special type of FPC (explained [below](#fee-paying-contract)) that can be used to pay for account deployment and regular txs.
+Note: the contract alias below is that of a sponsored fee paying contract.
+
+#include_code fpc-sponsored yarn-project/cli-wallet/test/flows/sponsored_create_account_and_mint.sh bash
+
+:::Note
+In the sandbox, the sponsored FPC address is printed at the end of its initial logs.
+:::
+
+### Fee Juice from Sandbox Test accounts
+
+In the sandbox pre-loaded test accounts can be used to cover fee juice when deploying contracts.
+
+First import them:
+
+#include_code import-test-accounts yarn-project/cli-wallet/test/flows/basic.sh bash
```

Comment:
how do I mint my own fee juice for new accounts?

Reply:
Explained in bridging section (which specifies `--mint`) parameter